sklearn/metrics/tests/test_classification.py

```diff
@@ -179,9 +180,9 @@ def test_classification_report_output_dict_empty_input():
             "support": 0,
         },
         "weighted avg": {
-            "f1-score": 0.0,
-            "precision": 0.0,
-            "recall": 0.0,
+            "f1-score": np.nan,
+            "precision": np.nan,
+            "recall": np.nan,
```

Comment:
This case shows a different behaviour. IMO, this is a corner case and probably `np.nan` is more meaningful because the metric is not defined. However, we should acknowledge it the changelog as a change of behaviour.